runtime/near-vm-runner/src/tests.rs

```diff
@@ -8,7 +8,6 @@ pub(crate) mod test_builder;
 mod ts_contract;
 mod wasm_validation;
 
-#[cfg(all(feature = "near_vm", target_arch = "x86_64"))]
```

Comment:
Why is it needed?

Reply:
Oh, this, I can add back but I was getting build errors locally and rust-analyzer wasn't working.

Reply:
I think it's better to remove the feature compilation flag (keep the change) as I'm guessing this causes a problem for everyone using rust-analyzer, atleast in vscode. It doesn't cause too much of a difference in compiling.

Reply:
I'd make this a separate PR even if it's one line, because @near/contract-runtime should have much better understanding why it is there